javascript.js

```diff
@@ -55,6 +56,9 @@ const config = {
     'one-var': [ERROR, NEVER],
     'no-whitespace-before-property': ERROR,
 
+    // Trailing commas
+    'comma-dangle': [WARN, ALWAYS_MULTILINE],
```

Comment:
I'd actually set that to NEVER... I think most of our code does _not_ have dangling commas in multiline, so this will immediately incur a lot of changes in projects upgrading

Reply:
Hmm, but I think using `ALWAYS_MULTILINE` has more advantages to use, no?
Also, this change shouldn't require much time to fix all places in the code because starting lint checking with `--fix` should fix this problem automatically.

Look: https://github.com/wombat-tech/eslint-config/assets/42468607/a8cb29a2-579e-47a5-9077-62f9a99fe942

Reply:
The video is not finally uploaded, so you cannot watch it in the browser 🫠

Reply:
It's easy to fix, but one required big commit for all projects. potentially creating conflicts in PRs.

Please do a small poll in #tech in Slack as this would be a change affecting a lot of people.

Reply:
As discussed in the all hands, we will go with enforcing the trailing comma.